go/oasis-node/cmd/common/flags/flags.go

```diff
@@ -19,10 +19,12 @@ const (
 	// CfgConsensusValidator is the flag used to opt-in to being a validator.
 	CfgConsensusValidator = "consensus.validator"
 
-	cfgVerbose = "verbose"
-	cfgForce   = "force"
-	cfgRetries = "retries"
-	cfgEntity  = "entity"
+	cfgVerbose       = "verbose"
+	cfgForce         = "force"
+	cfgRetries       = "retries"
+	cfgEntity        = "entity"
+	cfgSigner        = "signer"
+	cfgFileSignerDir = "signer.file.dir"
```

Comment:
Unsure as to whether I should include this flag, or just reuse the `entity` flag.